internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -724,6 +737,12 @@ func resourceKubernetesClusterNodePoolUpdate(d *pluginsdk.ResourceData, meta int
 		props.EnableFIPS = pointer.To(d.Get("fips_enabled").(bool))
 	}
 
+	if d.HasChange("gpu_driver") {
+		props.GpuProfile = &agentpools.GPUProfile{
+			Driver: pointer.To(agentpools.GPUDriver(d.Get("gpu_driver").(string))),
+		}
+	}
```

Comment:
This is marked as a `ForceNew` property in the schema, so should not be included in the update function (since it will destroy and recreate based on this flag, not update). If the property can in fact be changed, can we update the schema to reflect this? Can you confirm which is correct?